examples/pes/instanton/ch4h-cbe-rate/instanton/20/inst.instanton_FINAL_forces_7.xyz

```diff
@@ -0,0 +1,160 @@
+H 2.517593911122162e-05 0.0005168317829262747 2.4212715166171957e-05
+H 0.000992760929019454 0.0011650767517412675 0.0009946713064312718
+6
+CELL(abcABC):  200.000000 200.000000 200.000000 90.000000 90.000000 90.000000 cell{atomic_unit}  Traj: positions{atomic_unit}   Bead:       15
+H 0.004768111833142186 0.005595790775192454 0.004777385986809435
+C -0.006847234906848242 -0.008035806675632529 -0.006860530632479821
+H 5.009979098247185e-05 8.723020639145317e-05 0.0005295151095374706
+H 0.0005294770399899562 8.625318237420741e-05 5.22380083634033e-05
+H 2.644921504035835e-05 0.0005377388090366253 2.5448243512471432e-05
+H 0.0014730970165910404 0.0017287936804333268 0.001475943300910387
+6
+CELL(abcABC):  200.000000 200.000000 200.000000 90.000000 90.000000 90.000000 cell{atomic_unit}  Traj: positions{atomic_unit}   Bead:       16
+H 0.004593158814403253 0.005390466606414178 0.004602090253325031
+C -0.007051644246303112 -0.008275698110127294 -0.007065336937728261
+H 5.179726092485736e-05 8.992194766577198e-05 0.0005430092597791258
+H 0.0005429691196656705 8.892136471594368e-05 5.398946290036121e-05
+H 2.756491146449491e-05 0.0005515149337220748 2.654023112391712e-05
+H 0.0018361541453959516 0.0021548732742626697 0.0018397077305998264
+6
+CELL(abcABC):  200.000000 200.000000 200.000000 90.000000 90.000000 90.000000 cell{atomic_unit}  Traj: positions{atomic_unit}   Bead:       17
+H 0.004460353508761727 0.005234607031168181 0.004469025088882361
+C -0.0071867512274614844 -0.00843425748686677 -0.0072007063645074245
+H 5.304621741863968e-05 9.181628235488402e-05 0.0005514849632515251
+H 0.0005514430911901513 9.080129426131121e-05 5.527193147614184e-05
+H 2.8457508571833042e-05 0.0005601992258963406 2.7418733949957638e-05
+H 0.0020934508848657885 0.0024568336476349373 0.002097505663600785
+6
+CELL(abcABC):  200.000000 200.000000 200.000000 90.000000 90.000000 90.000000 cell{atomic_unit}  Traj: positions{atomic_unit}   Bead:       18
+H 0.004372285516973662 0.005131250963019696 0.004380784834889795
+C -0.007268636770252356 -0.008530357104019615 -0.007282751024462185
+H 5.386794899031599e-05 9.302221770646212e-05 0.0005563807026209844
+H 0.0005563376204165138 9.199913053592466e-05 5.6112786639417316e-05
+H 2.9078350838318553e-05 0.0005652327439431559 2.803183796196151e-05
+H 0.0022570673663402374 0.0026488520099565704 0.002261440862350028
+6
+CELL(abcABC):  200.000000 200.000000 200.000000 90.000000 90.000000 90.000000 cell{atomic_unit}  Traj: positions{atomic_unit}   Bead:       19
+H 0.0043286140005704965 0.005079998355306259 0.004337027947887151
+C -0.0073072203232715785 -0.00857563813294604 -0.007321409550842261
+H 5.427557847603736e-05 9.360853203688178e-05 0.0005586081874842108
+H 0.0005585644724526162 9.258181998816893e-05 5.6529042558040026e-05
+H 2.9396246548074597e-05 0.0005675287517714622 2.834636969595294e-05
+H 0.0023363700141221244 0.0027419206460876917 0.0023408979754613313
```

Comment:
Do we need this file in the main repo? What is its purpose?

Reply:
No, we don't. I will delete it.